src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int figure = random.nextInt(5);
+        int firstParametr = random.nextInt(60) + 1;
+        int secondParametr = random.nextInt(70) + 1;
+        int thirdParametr = random.nextInt(80) + 1;
+        String color = new ColorSupplier().getRandomColor();

```

Comment:
Leta make ColorSupplier as clas level variable not to create instance of ColorSupplier every time we cal getRandomFigure() method